casa_formats_io/casa_low_level_io/table.py

```diff
@@ -307,7 +307,7 @@ def read(cls, filename, endian='<'):
 
         return table
 
-    def as_astropy_table(self, data_desc_id=None):
+    def as_astropy_table(self, data_desc_id=None, all_ddids=False):
```

Comment:
Maybe just make it so you can set ``data_desc_id='all'``?

Reply:
done